examples/gitops/argocd-codecommit/repo-cc.tf

```diff
@@ -0,0 +1,98 @@
+resource "aws_codecommit_repository" "workloads_repo_cc" {
+  repository_name = "eks-blueprints-workloads-cc"
+  default_branch  = "main"
+}
+
+resource "aws_iam_user" "argocd_user" {
+  name = "argocd-cc"
+}
+
+resource "aws_iam_service_specific_credential" "argocd_codecommit_credential" {
+  service_name = "codecommit.amazonaws.com"
+  user_name    = aws_iam_user.argocd_user.name
+}
+
+resource "aws_iam_user_policy" "argocd_user_codecommit_ro" {
+  name = "argocd-user-codecommit-ro"
+  user = aws_iam_user.argocd_user.name
+
+  policy = jsonencode({
+    Version : "2012-10-17",
+    Statement : [
+      {
+        Effect : "Allow",
+        Action : "codecommit:GitPull",
+        Resource : aws_codecommit_repository.workloads_repo_cc.arn
+      }
+    ]
+  })
+}
+
+resource "aws_iam_role" "iam_for_lambda" {
```

Comment:
All of the lambda resources can be removed - not required for this example, ArgoCD will sync